apps/prechain/src/components/browse-screen.js

```diff
@@ -1094,9 +1098,11 @@ const FeedTabContent = React.memo(({ visible }) => {
 const ProposalItem = React.memo(({ proposalId }) => {
   const proposal = useProposal(proposalId);
   const { displayName: authorAccountDisplayName } = useAccountDisplayName(
-    proposal.proposer?.id
+    proposal?.proposer?.id
   );
 
+  if (!proposal) return null;
```

Comment:
When would this happen? Feels like this would better be handled by a parent (if at all) to not hide bugs.